lib/oli/interop/ingest.ex

```diff
@@ -338,7 +338,8 @@ defmodule Oli.Interop.Ingest do
         Oli.Authoring.Course.create_project(title, as_author, %{
           description: Map.get(project_details, "description"),
           legacy_svn_root: Map.get(project_details, "svnRoot"),
-          customizations: custom_labels
+          customizations: custom_labels,
+          required_survey_resource_id: Map.get(project_details, "required_student_survey")
```

Comment:
The value of "required_student_survey" must be read, and if not `nil`, mapped from the string id to the actual Torus database id of that resource.  This ID won't exist into all pages have been ingested and we know what the mapping of string ids to actual Torus Ids are.  So we probably can't set `required_survey_resource_id` here, it will need to be done as an update after all pages are processed.